pkg/collector/corechecks/snmp/status/status.go

```diff
@@ -0,0 +1,61 @@
+package status
```

Comment:
**🥜 nitpick:** ‏package comment and copyright are missing

Reply:
changed in this commit https://github.com/DataDog/datadog-agent/pull/25233/commits/97230d1ccffb1fff7226f1c4f502e6197165147b